### PR TITLE
set patronictl config file

### PR DESCRIPTION
**Pull request by jleni**
#### What this PR does / why we need it

Points patronictl to the correct config file.
This means that people can run the CLI without using `-c /etc/timescaledb/patroni.yaml`

#### Which issue this PR fixes

Minimizes the risk of mistakes such as

#179
#347
#418 

#### Checklist

- [x] Chart Version bumped
- [x] [CLA signed](https://cla-assistant.io/timescale/helm-charts)

### CODE REVIEW

File: charts/timescaledb-single/Chart.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.30.0
+version: 0.30.1
```
Comment (nhudson) [inline charts/timescaledb-single/Chart.yaml:7]: LGTM!  Thanks for the PR.  

The only nit is updating this to `0.31.0` when you can.

**Review by nhudson — CHANGES_REQUESTED** [review]

File: charts/timescaledb-single/Chart.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.30.0
+version: 0.31.0
```
Comment (nhudson) [inline charts/timescaledb-single/Chart.yaml:7]: Sorry this now needs to be `0.33.0`.  There have been a few merges since my last comment.

**Review by nhudson — CHANGES_REQUESTED** [review]

**Comment by jleni** [comment]
Yes, I just noticed! sorry about that.
Rebased and updated

**Comment by jleni** [comment]
sorry the rebased removed a line. Fixing that now

File: charts/timescaledb-single/Chart.yaml [hunk 3]
```diff
@@ -4,9 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.32.0
-icon: https://cdn.iconscout.com/icon/free/png-256/timescaledb-1958407-1651618.png
```
Comment (nhudson) [inline charts/timescaledb-single/Chart.yaml:8]: No need to delete the `icon:` line here.

**Review by nhudson — CHANGES_REQUESTED** [review]

File: charts/timescaledb-single/Chart.yaml (hunk 3 above)
Comment (jleni) [reply charts/timescaledb-single/Chart.yaml:8]: yes, it was due to the rebased. Reverted

**Pull request merged**